closed/autoconf/custom-spec.gmk.in

```diff
@@ -138,6 +138,12 @@ endif
 ifeq (riscv64,$(OPENJDK_TARGET_CPU))
   ifeq (cross,$(COMPILE_TYPE))
     export SYSROOT_CFLAGS
+    export RISCV_CROSSTOOLS_PREFIX := riscv64-unknown-linux-gnu
+    # change the prefix of the cross toolchain if the corresponding
+    # software package is installed on the host system.
+    ifeq (install,$(RISCV_TOOLCHAIN_TYPE))
+       export RISCV_CROSSTOOLS_PREFIX := riscv64-linux-gnu
```

Comment:
* Please indent consistently (2 spaces per level).
* The default should be in the else branch.
* Sentences should begin with a capital.
```
    # Change the prefix of the cross toolchain if the corresponding
    # software package is installed on the host system.
    ifeq (install,$(RISCV_TOOLCHAIN_TYPE))
      export RISCV_CROSSTOOLS_PREFIX := riscv64-linux-gnu
    else
      export RISCV_CROSSTOOLS_PREFIX := riscv64-unknown-linux-gnu
    endif
```

Reply:
Updated the script as suggested above.

Reply:
As discussed with @keithc-ca offline, changed the script to only set the value when `RISCV_TOOLCHAIN_TYPE `is` install `with the default value of `RISCV_CROSSTOOLS_PREFIX `set up in OpenJ9.